### DISCUSSION TITLE
activeTintColor only changes the text color not the bottom bar

### DISCUSSION THREAD

**Discussion by dutchkillscreative** [react-navigation/react-navigation#10638] [Q&A] [answered]
Is there a way to change the color of the blue indicator bar below the tabs? I don't see any other color options for controlling this value.

[Screenshot](https://imgur.com/a/gctf37u)

**Comment by raajnadar** [comment, marked as answer]
Next time read the docs also try the search feature.

https://reactnavigation.org/docs/material-top-tab-navigator/#tabbarindicatorstyle

**Reply by dutchkillscreative** [reply]
I did both, it’s not the most clear indication that this is what that option is referencing. Thank you for the reply, nonetheless.

**Reply by raajnadar** [reply]
If that fixes the issue let me know we can mark this as solved.
